packages/authentication/src/definitions.ts

```diff
@@ -483,6 +483,17 @@ export interface FirebaseAuthenticationPlugin {
     eventName: 'authStateChange',
     listenerFunc: AuthStateChangeListener,
   ): Promise<PluginListenerHandle>;
+  /**
+   * Listen for changes to the signed-in user's ID token changes.
+   *
+   * **Attention:** This listener will not be triggered automatically upon ID token expiration.
+   *
+   * @since 6.2.0
```

Comment:
```suggestion
   * @since 6.3.0
```

The next release will be 6.3.0, see https://github.com/capawesome-team/capacitor-firebase/pull/743.